internal/pkg/cli/pipeline_init.go

```diff
@@ -181,6 +181,7 @@ func (o *initPipelineOpts) Execute() error {
 		}
 		log.Successf("Secret already exists for %s! Do nothing.\n", color.HighlightUserInput(o.GitHubRepo))
 	}
+	log.Successf("Created the secret %s for pipeline source stage\n", color.HighlightUserInput(secretName))
```

Comment:
Are we going to log success twice for a secret now if the secret already exists?

Reply:
Oh good call. I can fix that.

Reply:
resolved by ec4d058